src/messages.tsx

```diff
@@ -21,3 +21,9 @@ export const NO_SPEC_FOUND = (command: string) => `${FALLBACK_MESSAGES[command]}
 export const FALLBACK_MESSAGES: {[name: string]: string} = {
   validate: 'Unable to perform validation. Specify what AsyncAPI file should be validated.'
 };
+
+export const CONTEXT_NAME = 'context-name';
+
+export const SPEC_PATH = 'spec-path';
+
+export const CONTEXT = 'context';
```

Comment:
```suggestion
export const Arguments = {
  context_name: 'context-name',
  spec_path: 'spec-path',
  context: 'context'
}
```

I think exporting an `Argument` object would be good as they will not clash with other variables and it is easy to understand what these constants are.

Reply:
Generally in JS global variables should be written with capital letters:

```js
export const ARGUMENTS = {
  CONTEXT_NAME: 'context-name',
  SPEC_PATH: 'spec-path',
  CONTEXT: 'context'
}
```

Reply:
Thanks for review, have updated above changes. Please check.